tests/utils/test_shell_util.py

```diff
@@ -40,5 +40,48 @@ def test_shellquote(self):
         self.assertEqual("\'foo bar\'", shellutil.quote("foo bar"))
         self.assertEqual("'foo'\\''bar'", shellutil.quote("foo\'bar"))
 
+    def test_it_should_log_command_failures_as_errors(self):
+        return_code = 99
+        command = "exit {0}".format(return_code)
+
+        with patch("azurelinuxagent.common.utils.shellutil.logger.error") as mock_error:
+            shellutil.run_get_output(command)
+
+        mock_error.assert_called_once()
```

Comment:
If we wanted to be completely sure the message was only logged as an error, maybe we can assert that the logger.info wasn't called as well.